types/ckeditor__ckeditor5-minimap/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for @ckeditor/ckeditor5-minimap 32.0
```

Comment:
the latest version is 33
https://www.npmjs.com/package/@ckeditor/ckeditor5-minimap